packages/cli/configuration/fern/definition/docs.yml

```diff
@@ -142,7 +148,9 @@ types:
   TabbedNavigationItem:
     properties:
       tab: TabId
-      layout: list<NavigationItem>
+      layout:
```

Comment:
[not blocking] for my SA, why is `layout` not on the tab object? It seems tied to a tab and then you could make it so that you can't have both by making a union

Reply:
this is because tabs are specified at the top level. i'm not a huge fan of this personally but we can't change it now.